tests/test_tracking.py

```diff
@@ -28,7 +27,10 @@
 from thriftpy.transport import TServerSocket, TBufferedTransportFactory, \
     TTransportException, TSocket
 from thriftpy.protocol import TBinaryProtocolFactory
-
+from compatible.version_2.tracking \
+    import TTrackedProcessor as old_TTrackedProcessor, \
```

Comment:
Do not use `old_` as prefix.  `import TTrackedProcessor as TTrackedProcessorV2` is a lot better.